community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -327,20 +327,11 @@ def ignore_err(e) -> bool:
 
 def sync_placement_groups():
     """Delete placement policies that are for jobs that have completed/terminated"""
-    keep_states = frozenset(
```

Comment:
Why did you move it? 
When I said that "something belongs to util" I've meant the `get_jobs` function.